packages/react-native/ReactAndroid/src/main/java/com/facebook/react/views/view/ReactViewGroup.java

```diff
@@ -1113,7 +1223,7 @@ private boolean isFocusDestinationsSet() {
     return focusDestinations.length > 0;
   }
 
-  private boolean isTVFocusGuide() {
+  public boolean isTVFocusGuide() {
```

Comment:
```suggestion
  protected boolean isTVFocusGuide() {
```

Reply:
This is used in ReactViewManager class, thus it cannot be protected

Reply:
I think that since `ReactViewGroup` and `ReactViewManager` are in the same package, then `protected` should be ok? I'll have a look in Android Studio, maybe I missed something.

Reply:
So I tried to make it package-private and it seems to be working